meshtastic/mesh_interface_runtime/request_wait.py

```
@@ -0,0 +1,542 @@
+        self._lock = lock
+        self._get_response_handlers = get_response_handlers
+        self._get_wait_errors = get_wait_errors
+        self._get_wait_acks = get_wait_acks
+        self._get_active_wait_request_ids = get_active_wait_request_ids
+        self._get_retired_wait_request_ids = get_retired_wait_request_ids
+        self._get_acknowledgment = get_acknowledgment
+        self._get_timeout = get_timeout
+        self._retired_wait_ttl_seconds = retired_wait_ttl_seconds
+        self._ack_nak_handlers: dict[int, bool] = {}
+
+    def mark_ack_nak_handler(self, request_id: int, flag: bool = True) -> None:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)